### DISCUSSION TITLE
GitHub Login issue

### DISCUSSION THREAD

**Discussion by rangap08** [community/community#46782] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I created Github account in 2017. I want to recover my account or create a new account using the same email. And I have lost my password, 2FA code and recovery code. 
Could you please provide steps to login Github.

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @rangap08 

You can access this link below with account recovery instructions and try to recover:

https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials

Another alternative would be to submit a request to GitHub Support, in the link below:

https://support.github.com/contact/account
